test/verify/check-system-services

```diff
@@ -263,6 +263,11 @@ Unit=test.service
         b.go("/system/services#test.service")
         b.enter_page("/system/services")
 
+        b.click(".cockpit-logline:contains('WORKING')")
+        b.enter_page("/system/logs")
+        b.click("button:contains('Go to test.service')")
+        b.enter_page("/system/services")
```

Comment:
Could this check that it actually shows the test.service details, not the overview?

Reply:
... asserting `window.location.hash` should work?

Reply:
Good call, added